libzkbob-rs/Cargo.toml

```diff
@@ -24,7 +24,7 @@ hex = { version = "0.4.3", features = ["serde"] }
 
 [dependencies.libzeropool]
 git = "https://github.com/zkbob/libzeropool-zkbob"
-branch = "master"
+branch = "feature/compliance"
```

Comment:
may be revert this change?
 there seems to be nothing new

Reply:
Reverted in https://github.com/zkBob/libzkbob-rs/pull/46/commits/b0a5549b05eaa4878be2cb8a8de349d90e61c970